policy/modules/contrib/hg.te

```diff
@@ -0,0 +1,65 @@
+policy_module(hg, 1.0.0)
+
+########################################
+#
+# Declarations
+#
+
+## <desc>
+##	<p>
+##	Determine whether Mercurial CGI
+##	can access cifs file systems.
+##	</p>
+## </desc>
+gen_tunable(hg_cgi_use_cifs, false)
+
+## <desc>
+##	<p>
+##	Determine whether Mercurial CGI
+##	can access nfs file systems.
+##	</p>
+## </desc>
+gen_tunable(hg_cgi_use_nfs, false)
+
+apache_content_template(hg)
+apache_content_alias_template(hg, hg)
+
+########################################
+#
+# CGI policy
+#
+
+list_dirs_pattern(hg_script_t, hg_content_t, hg_content_t)
+read_files_pattern(hg_script_t, hg_content_t, hg_content_t)
+
+auth_use_nsswitch(hg_script_t)
+
+gen_require(`
+	type httpd_config_t;
+	type httpd_sys_script_exec_t;
+')
+list_dirs_pattern(hg_script_t, httpd_config_t, httpd_sys_script_exec_t)
```

Comment:
Types from other modules should not be referred to directly, but using existing interfaces or together with creating new ones.